build.gradle

```diff
@@ -157,7 +160,9 @@ dependencies {
     compile "org.json:json:${jsonVer}"
     compile "com.google.inject:guice:${guiceVer}"
     compile "com.google.inject.extensions:guice-assistedinject:${guiceVer}"
-    compile "org.glassfish.jersey.containers:jersey-container-jetty-http:2.23.2"
+    compile "org.glassfish.jersey.containers:jersey-container-jetty-http:${jerseyContainerVer}"
+    compile "org.eclipse.jetty:jetty-servlet:${jettyServletVer}"
+    compile "com.owlike:genson:${gensonVer}"
```

Comment:
I'm surprised Gson and Jackson doesn't work with Jersey. Do you remember the exact issue ? Let's work together to identify the problem, and solve it. I'll be surprised if Gson and Jackson doesn't work for this widely practiced use-case.